gkeys/gkeys/base.py

```diff
@@ -222,6 +222,11 @@ def _option_uid(parser=None):
         parser.add_argument('-u', '--uid', dest='uid', nargs='+', default=None,
             help='The user ID, gpg key uid')
 
+    @staticmethod
+    def _option_email(parser=None):
+        parser.add_argument('-E', '--email', dest='email', default=None,
+            help='Email parameter for spec-check')
```

Comment:
Don't specify an individual target action for these options.  They are listed in this one file for re-use among the different actions.  It is just that so far, the only consumer of this option is spec-check.  but that could easily change.

So the description should be generic about toggling email output.